contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpAPIConfig.java

```diff
@@ -44,6 +45,17 @@
 
   private final String postBody;
 
+  public enum allowedMethods {
```

Comment:
```suggestion
  public enum AllowedMethods {
```

Reply:
I think it's better to call its http methods, whether you allow them or not you decide in your code.

Reply:
Renamed to `HttpMethods`